projects/baby_driver/inc/gpio_set.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+// This module allows a specific gpio pin to be set to a desired state when a
+// babydriver CAN message with a specific ID is received, using the dispatcher.
+// Requires dispatcher, and gpio to be initialized.

```

Comment:
Nitpick: you should include all the transitive dependencies here (dispatcher's dependencies, and dispatcher's dependencies' dependencies, etc.) so that someone including this module knows exactly what they have to initialize.